.teamcity/common/extensions.kt

```diff
@@ -91,8 +91,34 @@ fun BuildType.applyDefaultSettings(os: Os = Os.linux, timeout: Int = 30, vcsRoot
             param("env.LC_ALL", "en_US.UTF-8")
         }
     }
+
+    publishConventionsPlugin(os)
+
+}
+
+
+private fun BuildType.publishConventionsPlugin(os: Os) {
```

Comment:
Why not just use an included build?

Reply:
We can't do that atm because included builds break the configuration cache.

I am working on restructuring/cleanup of our buildSrc. So that we can easily switch to included builds once they are supported. Maybe this should then also just become a buildSrc plugin for now and later we move all of them to included builds.
Or does buildSrc not work for setting plugins?